build/dsc.rb

```diff
@@ -4,11 +4,15 @@
 require 'pathname'
 require 'erb'
 require 'open-uri'
-require 'mof'
-require 'cim'
 require 'find'
 require 'logger'
-require 'charlock_holmes/string'
+
+# Bundler group 'build'
+if Bundler.rubygems.find_name('mof').any?
```

Comment:
Fair enough - I'm good to move forward on this as a stop gap and come back to it later if we need to enhance it. :)